functional-tests/features/matchers/match_current_date.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Matchers
+    As a kakunin user
+    I want to navigate to matcher page and match current date
+
+    Scenario: I want to match current date
+        Given I visit the "main" page
+        When I click the "matchersLink" element
+        Then the "matchers" page is displayed
+        And there is element "dateElement" with value "f:currentDate:{yyyy}-{MM}-{dd}"
```

Comment:
Are you sure that we should be using `curly braces` in the feature files?
currently: `"f:currentDate:{yyyy}-{MM}-{dd}"`
proposition: `"f:currentDate:yyyy-MM-dd"`

@sethii  What do you think?

Reply:
This is library property to pass format LDML tokens https://sugarjs.com/dates/#/Formatting

Reply:
Don't you have an established format od dates used in the project?  I mean I guess there's a dictionary that says "here we use date with format: YY-mm-dd" or whatever else.

Reply:
As far as I know, sugar has something like global date format (in some cases we changed it into DD/MM/YYYY). 

Maybe instead of providing format as a parameter, we could integrate it into global specified format ? This ways we will make sure that the date format is the same in every date related code.